packages/frontend/src/components/LimitButton/index.tsx

```diff
@@ -28,7 +28,7 @@ const LimitButton: FC<Props> = memo(
             <Popover
                 disabled={disabled}
                 opened={opened}
-                position="bottom"
+                position="top"
```

Comment:
why top? good question. popover does not overlap results table, you can still see headers and data, find it a better UX especially for sorting popover

Reply:
agree!